numpy/core/numeric.py

```diff
@@ -998,7 +1020,9 @@ def tensordot(a, b, axes=2):
            [4664., 5018.],
            [4796., 5162.],
            [4928., 5306.]])
-    >>> # A slower but equivalent way of computing the same...
+
+    A slower but equivalent way of computing the same...
```

Comment:
Does this maybe break the doc generation?

Reply:
You can check this all locally with `spin docs` and `spin check-docs`.

Reply:
I don't think this is a problem because there some codes below also written this way

Reply:
I know what happened. The point is this pull request is too old, and the ``numeric.py`` was moved from ``core`` folder ``to _core`` folder. That causes the conflict. I modified it and now it works.